### PR TITLE
Don't support ffwd over http anymore, use semantic-metrics 1.1.8 where the lib is not included.

### PR DESCRIPTION
**Pull request by klaraward**
Can not find any usages of the config `ffwd.type: http` in the Spotify code base, and I believe @lmuhlha said she wants to get rid of the ffwd-http-client library - https://github.com/spotify/apollo/pull/326#discussion_r563832060

Include the old version of `ffwd-http-client` lib to make sure downstream libraries get that as a transitive dependency and can use the shaded packages.

### CODE REVIEW

**Review by lmuhlha — APPROVED** [review]

**Comment by klaraward** [comment]
We still have a problem with using a new version of semantic-metrics, because it brings in the new ffwd-http-client without the shaded dependencies, that too many people depend on.

**Review by lepistone — APPROVED** [review]
Sounds great together with https://github.com/spotify/semantic-metrics/pull/108. Thank you!

File: .circleci/config.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
       - run:
           command: mvn -Pcoverage -Pmissinglink verify
           environment:
-            MAVEN_OPTS: -Xmx512m
+            MAVEN_OPTS: -Xmx768m
```
Comment (toweroy) [inline .circleci/config.yml:22]: Why is this increase required?

File: pom.xml [hunk 2]
```diff
@@ -162,19 +162,20 @@
             </dependency>
             <dependency>
                 <groupId>com.spotify.metrics</groupId>
-                <artifactId>semantic-metrics-core</artifactId>
+                <artifactId>semantic-metrics-bom</artifactId>
                 <version>${semantic-metrics.version}</version>
```
Comment (toweroy) [inline pom.xml:166]: If we will use the `semantic-metrics-bom` now, is this parameter required?

File: .circleci/config.yml (hunk 1 above)
Comment (klaraward) [reply .circleci/config.yml:22]: Failed because of OOM before

**Pull request merged**